### PR TITLE
Added an image file

### PR DESCRIPTION
**Pull request by kjvbedzra**

### CODE REVIEW

File: sima_web_api/api/sale/controllers.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def sale_list_create_new(current_user):
             sale_list_id=new_sale_list.id,
         )
         db.session.add(new_sale)
-      8  db.session.commit()
```
Comment (yeboah326) [inline sima_web_api/api/sale/controllers.py:173]: Please remove this, it has already been fixed

File: src/App.js [hunk 2]
```diff
@@ -33,9 +33,9 @@ function App() {
           <div className='container'>
             <Route exact path = '/' component={SplashPage}/>
             <Route exact path = '/login'component={LoginForm}/>
-            <Route exact path = '/stocks1'component={Stocks1}/>
```
Comment (yeboah326) [inline src/App.js:36]: Please add Barnes to review this section

**Review by yeboah326 — CHANGES_REQUESTED** [review]
Please add Barnes as a reviewer

File: src/App.js (hunk 2 above)
Comment (kjvbedzra) [reply src/App.js:36]: sure

Comment (kjvbedzra) [reply src/App.js:36]: sure

**Review by barnes-joseph — APPROVED** [review]

**Review by barnes-joseph — APPROVED** [review]

**Review by yeboah326 — APPROVED** [review]

**Pull request merged**